README.md

```diff
@@ -307,13 +314,19 @@ and run `./script/build.sh -DUSE_SYSTEM_BOOST=NO`
 Compiling On Windows
 --------------------
 
-Currently VegaStrike is not compiling on Windows. Any help will be appreciated to get it fix. For more information go [here](https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/53)
+Vega Strike is now compiling on Windows! If you want to compile it, try it out, and perhaps offer feedback, that would certainly be welcome. (Installer still pending.)
+
+To compile Vega Strike on Windows, start by installing Visual Studio 2019, if you haven't already. When selecting the Workloads and Components to install, include at least "C++ for Desktop"; "Python"; and a recent build of the Windows SDK. Probably Git and the GitHub for Windows extension also. Install the latest Visual Studio updates as well.
+
+Once Visual Studio finishes installing, and you have rebooted your computer, open a PowerShell window and run `script/bootstrap.ps1`. Once that finishes, run `script/build.ps1`.
+
+Assuming all the above steps succeed, you are now ready to run Vega Strike. Note that `vegasettings` is not currently building on Windows, so you will need to edit `vegastrike.config` manually as needed.
```

Comment:
I would  add a note that we do not yet have an installer available for Windows.

Reply:
@BenjamenMeyer There is a note about that further up. I can add one here too if needed.

Reply:
Moved note about Windows installer down a paragraph or two. Hopefully it's more noticeable now.